examples/api/python/id.py

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env python
+###############################################################################
+# Top contributors (to current version):
+#   Alex Ozdemir
+#
+# This file is part of the cvc5 project.
+#
+# Copyright (c) 2009-2021 by the authors listed in the file AUTHORS
+# in the top-level source directory and their institutional affiliations.
+# All rights reserved.  See the file COPYING in the top-level source
+# directory for licensing information.
+# #############################################################################
+#
+# A simple demonstration of the solving capabilities of the cvc5 sets solver
```

Comment:
This description should be updated for this example.

Reply:
Good catch. Fixed in c7a99b06dd.